plugins/backend/backstage-plugin-aws-auth/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@roadiehq/backstage-plugin-aws-auth",
+  "version": "0.3.4",
```

Comment:
Does this need to be bumped?

Reply:
Actually we won't be publishing it atm as there no real changes, so it can be bumped when changelog is added, or when real changes does take place :)